src/sentry/api/urls.py

```diff
@@ -1104,6 +1105,12 @@
         name='sentry-api-0-sentry-app-details'
     ),
 
+    url(
+        r'^sentry-app-installations/(?P<uuid>[^\/]+)/authorizations/$',
+        SentryAppAuthorizationsEndpoint.as_view(),
+        name='sentry-api-0-sentry-app-authorizations'
```

Comment:
should this be 
```suggestion
        name='sentry-api-0-sentry-app-installations-authorizations'
```
?

Reply:
I decided to remove the `installations` from it. Seems to convey the same idea without it without the extra word.